manager/integration/tests/test_ha.py

```diff
@@ -1295,7 +1296,29 @@ def test_volume_reattach_after_engine_sigkill():  # NOQA
     7. Check md5sum of the data in the Pod.
     8. Check if data can be still written to the volume.
     """
-    pass
+    data_path1 = "/data/test1"
+    data_path2 = "/data/test2"
+
+    pod_name, _, _, md5sum = \
+        prepare_pod_with_data_in_mb(client, core_api,
+                                    pod_make,
+                                    volume_name,
+                                    data_size_in_mb=DATA_SIZE_IN_MB_1,
+                                    data_path=data_path1)
+
+    crash_engine_process(client, core_api, volume_name)
+
+    wait_for_volume_detached(client, volume_name)
+    wait_for_volume_healthy(client, volume_name)
+
+    wait_for_pod_restart(core_api, pod_name)
+    wait_for_pod_remount(core_api, pod_name)
+
+    res_md5sum = get_pod_data_md5sum(core_api, pod_name, data_path1)
+    assert md5sum == res_md5sum
+
+    write_pod_volume_random_data(core_api, pod_name,
```

Comment:
just add a final check to make sure the data was correct for the last write.

Reply:
fixed